### PR TITLE
remove version form package.json and bower.json as we do not update i…

### PR DESCRIPTION
**Pull request by VladDubrovskis**
…t for the last N versions. Keeps in line with other Origami components
Also removes the gulpfile.js as only purpose for it was to do github releases :)

/cc @adgad @pornel @leggsimon @andrewgeorgiou1981

### CODE REVIEW

**Comment by VladDubrovskis** [comment]
just realised I still need to remove all the unnecessary dependencies

File: package.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "o-ads",
-  "version": "3.0.1",
```
Comment (pornel) [inline package.json]: I'm afraid that one is mandatory: https://docs.npmjs.com/files/package.json#version

**Comment by VladDubrovskis** [comment]
https://github.com/npm/npm/issues/8512 - only required if published for public npm? Or am I missing something?

**Comment by VladDubrovskis** [comment]
will remove it from package.json as we do not use it in other Origami components. But will bring it up in Origami spec issue queue.

**Comment by leggsimon** [comment]
👍

**Pull request merged**